.github/workflows/cijoe_packages.yml

```diff
@@ -52,7 +52,15 @@ jobs:
 
     - name: Run it!
       run: |
-        cijoe --monitor -l
+        coverage run\
+          --data-file=cijoe-output/.coverage\
+          --rcfile=repository/.coveragerc\
```

Comment:
How about placing the coverage files in their own artifact? Currently, downloading the cijoe-report also includes the coverage info, which might give the impression that including coverage info is a standard cijoe behavior